### ISSUE TITLE
Remove native portion

### ISSUE THREAD

**Issue by ened** [YoussefKababe/pinch_zoom_image#7]
Is it possible to remove the native code portion altogether and use Flutters `SystemChrome` methods?

Besides, the Kotlin version is out of date again:
```
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.0 and higher.
The following dependencies do not satisfy the required version:
project ':pinch_zoom_image' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71
```

Highly suggest to move this back to Java..

**Comment by leodr** [comment]
Yes it is possible. Just clone the code into your own project and replace the channel.invokeMethod("hideStatusBar"); callback with SystemChrome.setEnabledSystemUIOverlays([]); and channel.invokeMethod("showStatusBar"); with SystemChrome.setEnabledSystemUIOverlays( [SystemUiOverlay.bottom, SystemUiOverlay.top]);

**Comment by MuhammamdArslanKhan** [comment]
Kindly help me, have anybody tried above solution? is it working?
mine is same issue.
`FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.0 and higher.
The following dependencies do not satisfy the required version:
project ':pinch_zoom_image' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
  Command: D:\PATH\android\gradlew.bat app:properties

Please review your Gradle project setup in the android/ folder.`

**Comment by leodr** [comment]
Well, yes I did try the above solution. It works fine, doesn't act as smooth tho. You will always have these build errors, unless the creator updates the package, which I think will unfortunately not happen.

**Comment by huobazi** [comment]
i also got this issues

```
Launching lib/main.dart on Android SDK built for x86 in debug mode...
Initializing gradle...
Resolving dependencies...
Running Gradle task 'assembleDebug'...

FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.0 and higher.
The following dependencies do not satisfy the required version:
project ':pinch_zoom_image' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
Finished with error: Gradle task assembleDebug failed with exit code 1

```

**Comment by huobazi** [comment]
https://github.com/YoussefKababe/pinch_zoom_image/pull/11 works fine for me

**Comment by furkankurt** [comment]
I created new repo. Owner of this project don't accept pull requests. You can use mine.
https://pub.dartlang.org/packages/pinch_zoom_image_last
If there is an issue, you can open an issue of my github repo.

**Comment by shinriyo** [comment]
this?
```
 [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: MissingPluginException(No implementation found for method showStatusBar on channel pinch_zoom_image)
```
